src/service/controllers/voice.js

```diff
@@ -128,6 +155,7 @@ exports.changeState = (req, state, options) => {
 exports.handleCall = (req, res) => {
   res.set({ "Content-type": "application/json" });
   var userName = auth.userByCode(req);
+  const user = conf.userByName(userName);
   var call = calls[req.params.callid];
   var agentCall;
```

Comment:
@yuriy-g 
I've seen only one issue, here, **agentCall** is undefined, and later, if the **call** is undefined, it is never initialized.
Meaning **agentCall.<property>** causes issues.